examples/triton_gpt2/README.ipynb

```diff
@@ -9,17 +9,20 @@
     "\n",
```

Comment:
<p>Nice one on adding the <code>default</code>  namespace explicitly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>